### ISSUE TITLE
Login with phone number

### ISSUE THREAD

**Issue by anandology** [anandology/cleansweep#89]
Some people may not have gmail/facebook account. It might be good idea to have ability to login via entering a phone number and password. Password could be set once or sent everytime via OTP.

@charanjeetaap do you think this is an important feature to have?

**Comment by charanjeetaap** [comment]
Login using Phone number(through OTP etc.) we can give, but i think we should give login account to only those volunteers who have their email id in the system.

**Comment by RaghavPro** [comment]
Yes. Manual registration should be there. I'm one of those who don't use Facebook and Gmail.
